os-logging/module.yaml

```diff
@@ -4,4 +4,3 @@ version: '1.0'
 description: os-logging script package.
 execute:
 - script: configure.sh
-  user: '185'
```

Comment:
Why was this removed?

Reply:
@rcernich because this script must be run as `root` as it executes `chown` command

Reply:
It was doing that before.  Is this because LAUNCH_DIR is not owned by jboss user?

Reply:
@rcernich I don't know how come it works for other dependencies but `chown jboss:root <file>` can't work without root permissions UNLESS the ownership was already `jboss:root` meaning that nothing was changed